setup.py

```diff
@@ -1,12 +1,16 @@
 from setuptools import setup
 
+#bring in __version__ from source
+#per https://stackoverflow.com/a/17626524
+#and https://stackoverflow.com/a/2073599
+with open('lineman/version.py') as ver: exec(ver.read())
```

Comment:
Oh we are evaling the code here too. 

Hmm... I'm going to have to think about this pattern and see how I feel about it. I think it's fine but my opinion hasn't settled either way yet